src/pages/learn/javascript/codemods-with-babel-plugins/index.mdx

```diff
@@ -352,7 +352,7 @@ Let's assume that the example code is something like this:
 
 ```jsx
 import Egghead from 'egghead'
-;<Egghead video={src} />
```

Comment:
the smallest of typos!

Reply:
You need to sign the [Contributor License Agreement.](https://github.com/eggheadio/egghead-next/pull/273#issuecomment-757013755)